source/ci_source/ci_source_helpers.js

```diff
@@ -11,7 +11,7 @@ import type { Env } from "./ci_source"
 */
 export function ensureEnvKeysExist(env: Env, keys: string[]): boolean {
   const hasKeys = keys.map((key: string): boolean => {
-    return env.hasOwnProperty(key) && env[key].length > 0
+    return env.hasOwnProperty(key) && env[key] != null && env[key].length > 0
```

Comment:
The Travis (and now Jenkins) CI source tests set an `env[key]` to null in the tests (our way of saying the key exists but there is no value for that key). This function was attempting to access `.length` on null, which throws a `TypeError`, so I believe this null/undefined check is useful here in case the `env` object has a supplied property but its value is falsy.